cheat/cheatsheets/git

```diff
@@ -17,9 +17,16 @@ git commit -m "Your commit message"
 # To edit previous commit message
 git commit --amend
 
+# Git commit in the past
+git commit --date="`date --date='2 day ago'`"
```

Comment:
Oh, I didn't know that. It's working for me too (using 2.4.3) If someone could confirm that it works on old versions too, I'll just make the change.